homeassistant/config.py

```diff
@@ -233,21 +233,67 @@ def create_default_config(config_dir, detect_location=True):
 
             config_file.write(DEFAULT_CONFIG)
 
+        # Check for existing secrets file.
+        # If it exists, back it up before recreating it.
+        if os.path.isfile(secret_path):
+            backup_secret_path = "{}.{}.bak".format(
+                secret_path,
+                int(date_util.as_timestamp(date_util.now()))
```

Comment:
For humans it would be easier to read if we are not use `epoch` but an ISO 8601 timestamp.